osu.Game/Online/API/Requests/SearchBeatmapSetsRequest.cs

```diff
@@ -62,4 +73,38 @@ public enum BeatmapSearchCategory
         [Description("My Maps")]
         Mine,
     }
+
+    public enum BeatmapSearchGenre
```

Comment:
Seeing the gap at id 8 I'd probably prefer this enum to have explicit values assigned everywhere instead of just for 9.